requirements.txt

```diff
@@ -4,3 +4,4 @@
 
 msgpack-python==0.4.8
 ujson==1.35
+email_validator
```

Comment:
requirements should always be pinned with a version.

Reply:
You also need to add `email_validator` to setup.py and add documentation both:
* on installing with `email_validator` (I think `email_validator` should be an optional dependency)
* and, a note saying email validator is used by `validate_email`